Checkout.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name = 'Checkout'
-  s.version = '3.5.3'
+  s.version = '4.0.0'
```

Comment:
I can see on this folder we have 2 podspec files Checkout.podspec and Frames.podspec. Do we need to update Frames.podspec as well ?

Reply:
Frames.podspec is also updated